src/core/positioning/internalgnssreceiver.h

```diff
@@ -31,6 +31,9 @@ class InternalGnssReceiver : public AbstractGnssReceiver
     explicit InternalGnssReceiver( QObject *parent = nullptr );
     ~InternalGnssReceiver() override = default;
 
+  public slots:
+    QAbstractSocket::SocketState socketState() override;
+
```

Comment:
```suggestion
```

This can be completely left away and rely on the default implementation of AbstractGnssReceiver